inference-engine/tests/functional/inference_engine/ir_serialization/exec_graph.cpp

```diff
@@ -109,7 +109,7 @@ class ExecGraphSerializationTest : public ::testing::Test {
     }
 };
 
-TEST_F(ExecGraphSerializationTest, DISABLED_ExecutionGraph_CPU) {
+TEST_F(ExecGraphSerializationTest, ExecutionGraph_CPU) {
```

Comment:
Why this tests is in `inference_engine`? I offer to move it to CPU func tests

Reply:
I propose to solve this question separately. The current PR only enables some tests.

Reply:
The issue of moving the test is not connected to this PR.
This code change is covered by https://github.com/openvinotoolkit/openvino/pull/3101
@ilyachur 
Please go ahead with that change.